src/components/PublicationRequestSections/RequestInfo/RequestInfo.js

```diff
@@ -57,13 +57,13 @@ const RequestInfo = ({ request }) => {
         <Col xs={3}>
           <KeyValue label={<FormattedMessage id="ui-oa.publicationRequest.externalRequestIds" />}>
             <div>
-              <ul>
-                {request?.externalRequestIds?.length ?
-                  request?.externalRequestIds.map(requestId => <li key={requestId.id}>{requestId.externalId}</li>)
-                  :
-                  <NoValue />
-                }
-              </ul>
+              {request?.externalRequestIds?.length ?
+                <ul>
+                  {request?.externalRequestIds?.map(requestId => <li key={requestId?.id}>{requestId?.externalId}</li>)}
```

Comment:
Don't need to optionally chain inside here because you already know there are externalRequestIds, and inside the map you know that requestId is not null/undefined